lib/python/rosie/ws.py

```diff
@@ -189,33 +225,114 @@ def _render(self, all_revs=0, data=None, filters=None, s=None):
             filters=filters,
             s=s,
             data=data)
+        )
+
+
+class HelloHandler(RosieDiscoService):
```

Comment:
I've suggested a change here, and committed it - I didn't mean to, and I'm a tad surprised that i was able to do it - I wanted to offer you some changes you could choose to merge or not!

The changes are
```python
from getpass import getuser
if get_user():
  data = self.HELLO % get_user()

```

Reply:
You do appear to have done so & I am also confused as to how that has been allowed! But that is no worries at all. I am unsure whether it will remain after I next push my local branch, but I will ensure the underlying problem as outlined is fixed.

Excellent point. I am not sure at this stage, but doubtful, if there is a need to import ``getuser``, but 'll play around to ensure the current user gets assigned properly via some means.

Reply:
I've just force pushed my own branch to overwrite this as I suspect the Tornado-based approach is more standard & wouldn't require the extra import. In case you were interested in how GitHub/``git`` managed the incorporation of this commit on my end, upon trying to push I got warned as follows:

```
To github.com:sadielbartholomew/rose.git
 ! [rejected]          HEAD -> i2271-rosie-disco-migrations (fetch first)
error: failed to push some refs to 'git@github.com:sadielbartholomew/rose.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```